### PR TITLE
 OP-373 Increase coverage of admission package

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Coverage before:
image

Coverage after:
image

### CODE REVIEW

**Comment by dbmalkovsky** [comment]
The reason for the last commit is so I can try to figure out why two tests fail in CI build and not locally.

**Comment by mwithi** [comment]
Hi @dbmalkovsky, thanks for your work.

> "Chore Increase coverage of admission package"

This is more than a "chore" since we have an issue for that: OP-147. Could you please title accordingly?

**Comment by dbmalkovsky** [comment]
Is there anything else that is needed for the PR? @mwithi

**Comment by mwithi** [comment]
@dbmalkovsky not very, I'm setting @emecas as a reviewer who can do it faster than me in these days!

**Comment by emecas** [comment]
@dbmalkovsky  ignored `org.isf.admission.test.testIoLoadAdmittedPatient()` test is passing locally, is that a renamed version of one of the 2 tests that failed in [this checks ](https://github.com/informatici/openhospital-core/runs/1459644474)?

```
Results :

Failed tests:   mgrLoadAdmittedPatient(org.isf.admission.test.Tests): (..)
  loadAdmittedPatient(org.isf.admission.test.Tests): (..)

Tests run: 464, Failures: 2, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:53 min
[INFO] Finished at: 2020-11-26T15:20:28Z
[INFO] ------------------------------------------------------------------------
```

**Comment by dbmalkovsky** [comment]
@emecas Given the method name (missing the "test" at the front it appears that you are looking at older code.

The method is
````
	@Ignore
	@Test
	@Transactional
	public void testIoLoadAdmittedPatient() throws Exception {
		int id = _setupTestAdmission(false);
		Admission admission = admissionIoOperation.getAdmission(id);
		assertThat(admission).isNotNull();
		assertThat(admissionIoOperation.loadAdmittedPatient(id)).isNotNull();
	}
````
and yes it was failing and that is why I added `@Ignore` to it.

File: src/test/java/org/isf/admission/test/Tests.java [hunk 1]
```diff
@@ -542,9 +474,560 @@ public void testIoDeletePatientPhoto() throws Exception {
...
+
+	@Ignore
+	@Test
+	@Transactional
+	public void testIoLoadAdmittedPatient() throws Exception {
+		int id = _setupTestAdmission(false);
+		Admission admission = admissionIoOperation.getAdmission(id);
+		assertThat(admission).isNotNull();
+		assertThat(admissionIoOperation.loadAdmittedPatient(id)).isNotNull();
+	}
```
Comment (emecas) [inline src/test/java/org/isf/admission/test/Tests.java:490]: Playing with this here: https://github.com/informatici/openhospital-core/pull/273
but tests is still failing: https://github.com/informatici/openhospital-core/pull/273/checks?check_run_id=1596783555

**Review by emecas — APPROVED** [review]
LGTM

thanks @dbmalkovsky

**Review by mwithi — APPROVED** [review]
LGTM
```
Results :

Tests run: 607, Failures: 0, Errors: 0, Skipped: 2
```

**Comment by emecas** [comment]
@mwithi  assertj upgrading to `3.18.1` in 1626cc0  is relate to a new constructor of `Condition` class that @dbmalkovsky  introduced on `org.isf.admission.test.testMgrValidate` ,  since it is not available in the default version `2.6.0` for Spring `4.3.28.RELEASE`

**Comment by mwithi** [comment]
Thanks @emecas!

**Pull request merged**